tools/pv_tree_generator/util.py

```diff
@@ -134,6 +134,32 @@ def _read_pop_obs_spec():
     return result
 
 
+def removeDuplicateStatsVar(stat_vars):
```

Comment:
please  add some comments on the logic and rational?

Reply:
Added